src/system.rs

```diff
@@ -105,6 +107,47 @@ impl SwapMemory {
     }
 }
 
+#[derive(Debug)]
+pub struct LoadAverage {
+    /// number of jobs in the run queue averaged over 1 minute
+    pub one: f32,
+
+    /// number of jobs in the run queue averaged over 5 minute
+    pub five: f32,
+
+    /// number of jobs in the run queue averaged over 15 minute
+    pub fifteen: f32,
+
+    /// current number of runnable kernel entities
+    pub runnable: i32,
+
+    /// total number of runnable kernel entities
+    pub total_runnable: i32,
+
+    /// pid for the most recently created process
+    pub last_pid: PID,
+}
+
+impl LoadAverage {
+    fn new(
```

Comment:
Since `fn new` isn't doing any calculations or deriving any values, probably better to not have it and just use a struct literal (as is done for Fd at https://github.com/borntyping/rust-psutil/blob/master/src/process.rs#L568).